expr/func_isnonnull.go

```diff
@@ -42,7 +42,7 @@ func (s *FuncIsNonNull) Exec(cache map[Req][]models.Series) ([]models.Series, er
 			Consolidator: serie.Consolidator,
 			QueryCons:    serie.QueryCons,
 		}
-		serie.Tags["isNonNull"] = "1"
+		transformed.Tags["isNonNull"] = "1"
```

Comment:
This is still the same map reference. I expect you will need to copy the tags to a new map, and add the new tags there.

Reply:
Is not modifying the original series important?

If so, should I also copy the value when I modify it on lines 48-50?

Reply:
Not modifying the original is important, as it is cached and could be used by other functions. 

On lines 48-50 the variable `p` is already a copy.